Changelog.md

```diff
@@ -1,3 +1,30 @@
+### 0.7.2 (2020-12-17)
+
+Features:
+
+* Applied the [Fixed-width SIMD](https://github.com/webassembly/simd) proposal.
+  * Please see the [document](https://github.com/second-state/SSVM/blob/master/doc/simd.md) for detail.
+* Supported toggle enable or disable different proposals.
+* Added the option to enable or disable compiling tools.
+
+Fixed issues:
+
+* Fixed `call_indirect` table index checking in the validation phase.
+
+Tools:
+
+* Deleted `ssvmr` tool.
```

Comment:
* Deprecated `ssvmr` tool, since the functionalities are the same as `ssvm` tool.